src/openloco/windows/industry_list.cpp

```diff
@@ -1,16 +1,1323 @@
+#include "../audio/audio.h"
+#include "../config.h"
+#include "../date.h"
+#include "../graphics/colours.h"
+#include "../graphics/image_ids.h"
+#include "../industrymgr.h"
+#include "../input.h"
 #include "../interop/interop.hpp"
+#include "../localisation/FormatArguments.hpp"
+#include "../objects/cargo_object.h"
+#include "../objects/interface_skin_object.h"
+#include "../objects/objectmgr.h"
 #include "../openloco.h"
 #include "../ui/WindowManager.h"
+#include "../ui/scrollview.h"
+#include "../widget.h"
 
 using namespace openloco::interop;
 
 namespace openloco::ui::windows::industry_list
 {
+    static loco_global<uint32_t, 0x00E0C39C> dword_E0C39C;
+    static loco_global<uint8_t, 0x00E0C3D9> byte_E0C3D9;
+    static loco_global<uint8_t, 0x00525FC7> byte_525FC7;
+    static loco_global<uint16_t, 0x00523390> _toolWindowNumber;
+    static loco_global<utility::prng, 0x00525E20> _prng;
+    static loco_global<uint32_t, 0x00E0C394> _dword_E0C394;
+    static loco_global<uint32_t, 0x00E0C398> _dword_E0C398;
+    static loco_global<uint16_t, 0x00E0C3C6> word_E0C3C6;
+    static loco_global<ui::WindowType, 0x00523392> _toolWindowType;
+    loco_global<uint8_t, 0x00508F14> _screen_flags;
+    loco_global<int8_t, 0x00F2533F> _gridlines_state;
+
+    static loco_global<uint32_t[32], 0x00525E5E> currencyMultiplicationFactor;
+
+    namespace common
+    {
+        static const gfx::ui_size_t windowSize = { 600, 197 };
+        static const gfx::ui_size_t maxDimensions = { 600, 900 };
+        static const gfx::ui_size_t minDimensions = { 192, 100 };
+
+        static const uint8_t rowHeight = 10;
```

Comment:
rowHeight is just for the industry_list. The new_industry has a bigger rowHeight